system-design/youtube-start-seconds.md

```diff
@@ -0,0 +1,58 @@
+# YouTube Iframe API - startSecond
```

Comment:
Please use the appropriate markdown syntax for code: https://www.markdownguide.org/basic-syntax/#code

Please check for other occurrences

Reply:
This is a heading H1

Reply:
Please use codeblock for the API names: https://www.markdownguide.org/basic-syntax/#code